### ISSUE TITLE
gt accuracy

### ISSUE THREAD

**Issue by wmcnally** [facebookresearch/VideoPose3D#174]
Thank you for this great code. How come when I run
`$ python run.py -k gt -arc 3,3,3,3,3 -c checkpoint --evaluate pretrained_h36m_cpn.bin`

I get the following accuracy:
```
Protocol #1   (MPJPE) action-wise average: 42.6 mm
Protocol #2 (P-MPJPE) action-wise average: 32.8 mm
Protocol #3 (N-MPJPE) action-wise average: 40.4 mm
Velocity      (MPJVE) action-wise average: 2.09 mm
```
which is inconsistent with the paper for gt?

**Comment by dariopavllo** [comment]
Hi,

The pretrained model we provide was trained on CPN detections.
To reproduce the results relative to ground-truth keypoints, you need to retrain the model on ground-truth keypoints.

**Comment by wmcnally** [comment]
Got it. Thanks!
